dubbo-metrics/dubbo-metrics-metadata/src/test/java/org/apache/dubbo/metrics/metadata/MetadataMetricsCollectorTest.java

```diff
@@ -62,7 +63,7 @@ public void setup() {
         config.setName("MockMetrics");
 
         applicationModel.getApplicationConfigManager().setApplication(config);
-        applicationModel.getBeanFactory().getOrRegisterBean(MetricsDispatcher.class);
+        applicationModel.getBeanFactory().getOrRegisterBean(MetricsConstants.METRICS_DISPATCHER_NAME,MetricsDispatcher.class);
```

Comment:
Is there any purpose to name the bean registered by MetricsDispatcher separately? As far as I understand, this bean is unique

Reply:
It's a mistake, I will fix it later  :)